src/elements/OcFilterMenuItem.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <li>
+    <slot name="default">
+      <div class="uk-flex uk-flex-middle">
+        <oc-checkbox class="uk-margin-small-right" />
```

Comment:
- [ ] event when changing the checkbox needs to be emitted
- [ ] when clicking on the label usually a change on the corresponding checkbox is triggered